ggshield/cmd/iac/scan/diff.py

```diff
@@ -124,7 +85,9 @@ def iac_scan_diff(
             display_info(f"- {click.format_filename(filepath)}")
         display_info("")
 
-    current_ref = INDEX_REF if include_staged else "HEAD"
+    current_ref = current
+    if current_ref is None:
+        current_ref = INDEX_REF if include_staged else "HEAD"
```

Comment:
I prefer the following, not necessarily better:

```suggestion
    current_ref = current or (INDEX_REF if include_staged else "HEAD")
```

Reply:
IMHO we will gain some lines but lose readability. There are too much done on one line.